src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdater.java

```diff
@@ -48,29 +48,13 @@ protected void updateSingleEntry(BloggerEntry bloggerEntry, UpdateSummary update
             Blog bloggerWithSameJsonId = existingBloggerByJsonId.get();
             updateBloggerIfThereAreSomeChanges(bloggerEntry, updateSummary, bloggerWithSameJsonId);
         } else {
-            blogRepository.save(Blog.builder()
-                    .jsonId(bloggerEntry.getJsonId())
-                    .author(bloggerEntry.getName())
-                    .rss(StringUtils.lowerCase(bloggerEntry.getRss()))
-                    .url(syndFeedFactory.urlFromRss(
-                        bloggerEntry.getRss()).orElse(null)
-                    )
-                    .twitter(bloggerEntry.getTwitter())
-                    .dateAdded(nowProvider.now())
-                    .blogType(bloggerEntry.getBlogType())
-                    .active(true)
-                    .build());
-            updateSummary.recordCreated();
+            createNewBlogger(bloggerEntry, updateSummary);
         }
     }
 
     private void updateBloggerIfThereAreSomeChanges(BloggerEntry bloggerEntry,
                                                     UpdateSummary updateSummary,
                                                     Blog existingBlogger) {
-        bloggerEntry.setUrl(
```

Comment:
I was generally thinking about refactoring both `BloggersDataUpdater` and `BloggersDataFetcher` because they are doing some weird stuff. But i wanted to keep this PR clean and separate two different things into separate tasks.